boards/nucleo-f031k6/doc.txt

```diff
@@ -0,0 +1,2 @@
+/**
```

Comment:
why is this empty? Shouldn't it at least have the `@defgroup`?

Reply:
yes, true. I will fix it